janusgraph-core/src/main/java/org/janusgraph/graphdb/database/management/ManagementLogger.java

```diff
@@ -191,12 +208,24 @@ public void run() {
                         txStillOpen = true;
                     }
                 }
-                if (!txStillOpen) {
+                final JanusGraphManager jgm = JanusGraphManager.getInstance();
+                final boolean janusGraphManagerIsInBadState = null == jgm && action.equals(EVICT);
```

Comment:
If `jgm` is null couldn't that just mean that JanusGraph Server is not being used?

Reply:
The only way `action.equals(EVICT)` is if the user actively called `mgmt.evictGraphFromCache()` on their graph's mangementSystem, or if it was called by the ConfiguredGraphFactory [updateConfiguration](https://github.com/dpitera/janusgraph/blob/06d519e48dd64ae8df4dda5c1e94ca34d2eac250/janusgraph-core/src/main/java/org/janusgraph/core/ConfiguredGraphFactory.java#L203) or [removeConfiguration](https://github.com/dpitera/janusgraph/blob/06d519e48dd64ae8df4dda5c1e94ca34d2eac250/janusgraph-core/src/main/java/org/janusgraph/core/ConfiguredGraphFactory.java#L229). 

The issue in this scenario is indeed that a user is using the JanusGraph Server (more specifically configured to use the JanusGraphManager) for one of their JG nodes but not the other node (the one that _received_ this eviction request).